src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -152,14 +152,14 @@ public static WallType Wrap(Autodesk.Revit.DB.WallType ele, bool isRevitOwned)
 
         public static View3D Wrap(Autodesk.Revit.DB.View3D view, bool isRevitOwned)
         {
-            if (view.IsPerspective)
+            if (!view.IsTemplate)
```

Comment:
I think it is OK to ignore them. I think our users only cares about normal views (that may have a view template applied).